source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -176,6 +186,9 @@ You can select and initialise a specific component by using its `data-module` at
   </script>
 ```
 
+When initialised individually, errors are thrown rather than logged.
+You should check your application works without errors or some components will not work correctly.
```

Comment:
Cheers that’s loads simpler

Want to upgrade that “should” to a “must” in this case?

```suggestion
When initialised individually, errors are thrown rather than logged. You must check your application works without errors or some components will not work correctly.
```